.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: java
+
+dist: trusty
```

Comment:
Never mind.. unfortunately it seems trusty/14.04 is the latest Ubuntu release supported by Travis.

https://github.com/travis-ci/travis-ci/issues/5821